### PR TITLE
added a quickstart documentation

### PR DESCRIPTION
**Pull request by tibeer**
Started messing with Pulp and had a pretty simple issue (#531).
Thought it would be a good idea to create a quickstart so others do not fall for the same issue.

Signed-off-by: Tim Beermann <beermann@osism.tech>

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+Sync (again) AFTER distribution and publication
+--------------------------------------------------------------------------------
+
+If you need to sync after you have already distributed and published a repository,
+use the following commands to do so. The autopublish feature like in ``pulp_rpm`` is not implemented (yet).
+
+.. code-block:: bash
+
+   pulp deb repository sync --name "${name}" --mirror --remote "${name}"
+   pulp deb distribution update --name "${name}" --repository "${name}" --base-path "${name}"
```
Comment (mdellweg) [inline docs/quickstart.rst:48]: This is a no-op and should not be necessary.

Comment (tibeer) [reply docs/quickstart.rst:48]: What exactly should not be necessary?
1. pulp deb repository sync
2. pulp deb distribution update
3. both above

Comment (mdellweg) [reply docs/quickstart.rst:48]: Sorry, the distribution update is not needed, as the distribution is already set up for auto-distributing.

Comment (tibeer) [reply docs/quickstart.rst]: fixed.

Comment (quba42) [reply docs/quickstart.rst]: I may be mixing something up, but it was my understanding that there currently is no auto-distributing for pulp_deb...

**Comment by tibeer** [comment]
not stale

**Review by quba42 — APPROVED** [review]
First of all my apologies for taking so long to get around to this. These examples look valuable for users to have.

I propose the following for moving forward with this: Rather than endlessly going back and forth on minor details (documentation can always be tweaked further), I will merge this as is and may then add a small commit of my own making minor adjustments on top.

If you agree, could you rebase this onto latest main branch one more time to give the tests a chance to turn green? I will then merge it.

**Comment by tibeer** [comment]
> First of all my apologies for taking so long to get around to this. These examples look valuable for users to have.
> 
> I propose the following for moving forward with this: Rather than endlessly going back and forth on minor details (documentation can always be tweaked further), I will merge this as is and may then add a small commit of my own making minor adjustments on top.
> 
> If you agree, could you rebase this onto latest main branch one more time to give the tests a chance to turn green? I will then merge it.

@quba42 sure, done!

**Pull request merged**